src/Microsoft.AspNetCore.Razor.Language/RazorProjectFileSystem.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public abstract class RazorProjectFileSystem : RazorProject
```

Comment:
Thought it was fine if we still have all the methods return `RazorProjectitem` instead of `RazorProjectFileSystemItem`. If we went with the later idea we'd have to re-implement all of RazorProject. It wouldn't be too awful but I felt that `RazorProjectItem` is sufficient enough.

Reply:
I think you made the right choice.